explain-overview.md

```diff
@@ -6,500 +6,52 @@ aliases: ['/docs/dev/query-execution-plan/','/docs/dev/reference/performance/und
 
 # `EXPLAIN` Overview
```

Comment:
Because this is an `overview`, would it be better if we add links to the `EXPLAIN` series in this doc and explain a little bit about the series?

Reply:
Done.